kate_brownbill_dungon_fighters_v1.0.py

```diff
@@ -0,0 +1,838 @@
+                print(p2_name + " sent out " + act_f2.name)
+                time.sleep(4)
+                print("==========" + p2_name + "'s new fighter==========")
+                check_fighter(act_f2)
+                time.sleep(6)
+
+        #This section of code gets each player's move
+        turn1 = player1.get_turn(teem1, act_f1, act_f2, player2)
+        if turn1[0] == 2:
+            winner = -1
+            return winner
+        #time.sleep(3)
+        turn2 = player2.get_turn(teem2, act_f2, act_f1, player1)
+        #time.sleep(3)
+        #this handles the exiting
+        if turn1[0] == 2 or turn2[0] == 2:
+            winner = -1
+            return winner
+        elif winner == 0:
+            #print("==========Results==========")
+            
+            #this handles the swaping
+            if turn1[0] == 0:
+                print(p1_name + " withdrew " + act_f1.name)
+                act_f1 = turn1[2]
+                print(p1_name + " sent out " + act_f1.name)
+            if turn2[0] == 0:
+                print(p2_name + " withdrew " + act_f2.name)
+                act_f2 = turn2[2]
+                print(p2_name + " sent out " + act_f2.name)
+            #this handles the attacking
+            if turn1[0] == 1:
+                print(p1_name + "'s fighter " + act_f1.name + " used " + move_to_txt[turn1[1]])
+            if turn2[0] == 1:
+                print(p2_name + "'s fighter " + act_f2.name + " used " + move_to_txt[turn2[1]])
+            #this handles damage
+            fighters = do_attack(turn1[1], act_f1, turn2[1], act_f2)
+            teem1[teem1.index(act_f1)] = fighters[0] # updates the fighters in the list
+            teem2[teem2.index(act_f2)] = fighters[1]
+            player1.update_counter(turn2[1])
+            player2.update_counter(turn1[1])
+            time.sleep(5)
+ 
+ 
+def heal_team(team):
+    for i in team:
+        i.restore()
+    
+#Crall and shut down the portle to hell!
+def do_dungeon_crawl():
+    bf1 = Fighter([FRWD_SWNG, SHIELD_BASH, MAGIC_SWNG],"Bob", "A simpleminded fighter.", 5, 3, 2)
+    bf2 = Fighter([FIREBOLT, MAGIC_RFCT, SHIELD_BASH], "Magios", "A mage who weilds a protective sheild.", 4, 4, 2)
+    bf3 = Fighter([FRWD_SWNG, MAGIC_SWNG, MAGIC_RFCT], "Kyrin", "A fragile arcane rouge who deals devistaiting blows.",  3, 6, 1)
+    party = [bf1, bf2, bf3]
+    print("Welcome to dungeon crawl mode.")
+    print("Please select a name for your adventurer.")
+    name = raw_input()
+    player = Player(0, name, "The leader of a legendary party of adventurers.")
+    a1 = make_temple()
+    a2 = make_caverns()
+    a3 = make_hell()
+    world = [a1, a2, a3]
+    print("==========REMEMBER==========")
+    print("It is a good idea to constently")
+    print("check the INFO about both the moves")
+    print("the enemy fighter, and your team")
+    print("============================")
+    time.sleep(4)
+    for a in world:
```

Comment:
a and t are classic lazy variable names.  They don't give the reader any hint about what they represent.